api/src/unraid-api/graph/resolvers/array/parity.resolver.ts

```
@@ -0,0 +1,42 @@
+
+const pubSub = new PubSub();
+
+@Resolver(() => ParityCheck)
+export class ParityResolver {
+    @Query(() => [ParityCheck])
+    async parityHistory(): Promise<ParityCheck[]> {
+        throw new Error('Not implemented');
+    }
+
+    @Mutation(() => Object)
+    async startParityCheck(@Args('correct') correct: boolean): Promise<Object> {
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)